### PR TITLE
Examples: Use Webpack HTML plugin to build index.html, and fix npm script

### PR DESCRIPTION
**Pull request by asci**
## What
- Use `html-webpack-plugin` to build `index.html` page automatically
- fix for `npm run build` script

### CODE REVIEW

File: examples/multiple-apps/build/index.html [hunk 1]
```diff
@@ -12,9 +12,5 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.3/lodash.min.js"></script>
```
Comment (fahad19) [inline examples/multiple-apps/build/index.html]: if this file is generated by Webpack, let's delete it from the repo, and also ignore it from `.gitignore`.

**Comment by fahad19** [comment]
(I am taking care of the ESLint fails on Travis-CI on a separate branch)

**Comment by fahad19** [comment]
@asci: please update your branch from latest `master`, and the build error will go away.

File: examples/multiple-apps/build/index.html (hunk 1 above)
Comment (asci) [inline examples/multiple-apps/build/index.html]: done

File: examples/multiple-apps/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+build
```
Comment (fahad19) [inline examples/multiple-apps/.gitignore]: I think only `build/index.html` would be enough.

**Comment by fahad19** [comment]
one problem of deleting the entire `build` directory is, Webpack bundling would fail with an error for new users.

keep a file like `build/js/.gitkeep` to avoid this issue.

**Comment by jackTheRipper** [comment]
What about the other examples?

File: examples/multiple-apps/.gitignore (hunk 2 above)
Comment (alexmiranda) [inline examples/multiple-apps/.gitignore]: Probably it makes more sense to exclude the `build` folder altogether, unless you plan to have anything non-generated in that folder.

**Comment by asci** [comment]
The build time with CommonChunks plugin is too long. I reverted changes and provided another PR #200

**Pull request closed without merging**